packages/react-vapor/src/components/browserPreview/tests/BrowserPreview.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import {mount, ReactWrapper, shallow} from 'enzyme';
+import * as React from 'react';
+
+import {Tooltip} from '../../tooltip';
+import {BrowserPreview, BrowserPreviewProps} from '../BrowserPreview';
+
+describe('BrowserPreview', () => {
+    let component: ReactWrapper<BrowserPreviewProps>;
+    const mountWithProps = (props: BrowserPreviewProps) => {
+        component = mount(<BrowserPreview {...props} />, {
+            attachTo: document.getElementById('App'),
+        });
+    };
+
+    it('renders without errors', () => {
+        expect(() => {
+            shallow(<BrowserPreview />);
+        }).not.toThrow();
+    });
+
+    it('renders the specified header description as tooltip title', () => {
+        const headerDescription = '🥰';
+        mountWithProps({headerDescription});
+
+        expect(component.find(Tooltip).prop('title')).toEqual(headerDescription);
+    });
```

Comment:
Ah, since you're only using this method once I would directly mount in the test and remove the mountWithProps method:

```suggestion
    it('renders the specified header description as tooltip title', () => {
        const headerDescription = '🥰';
        component = mount(<BrowserPreview headerDescription={headerDescription} />);
        
        expect(component.find(Tooltip).prop('title')).toEqual(headerDescription);
    });
```